prov/efa/src/rxr/rxr.h

```diff
@@ -164,6 +164,7 @@ static inline void rxr_poison_mem_region(uint32_t *ptr, size_t size)
  * 60 - 63      provider specific
  */
 #define RXR_NO_COMPLETION	BIT_ULL(60)
+#define RXR_NO_COUNTER		BIT_ULL(61)
```

Comment:
Does `RXR_NO_COUNTER` have to be introduced here? Is `RXR_NO_COMPLETION` sufficient?

Reply:
it is used in `rxr_cq_handle_tx_completion`

Reply:
Yes, I see that. I'm not sure why the handshake packet needs this flag.

Reply:
@aolia-0303 there was some related change in rxr_cq_handle_tx_completion() that uses this flag need to be included in this PR.

Reply:
Makes sense, sorry for my confusion here. I think I just got hung up with the name - was taking "no counter" too literally and didn't see why it was related.